src/set_theory/game/ordinal.lean

```diff
@@ -27,6 +27,7 @@ open pgame
 
 local infix ` ≈ ` := equiv
 local infix ` ⧏ `:50 := lf
+local infix ` ≡r `:50 := relabelling
```

Comment:
Is there a way we can avoid having all these local infixes in all these files? I think we can make a locale but I don't know how.

Reply:
I agree that this does look like a good use case for a locale. You may have seen this already, but the docs for making localized commands are [here](https://leanprover-community.github.io/mathlib_docs/commands.html#localized%20notation). And of course feel free to ask on Zulip if you need help!